tests/test_qemu.py

```
@@ -21,9 +21,9 @@ def find_qemu():
     raise RuntimeError("QEMU not found. Set QEMU_BIN env var or add qemu-system-xtensa to PATH")
 
 QEMU = find_qemu()
-FLASH = os.environ.get("FLASH", "../build/merged_flash.bin")
+FLASH = os.environ.get("FLASH", "../build/qemu_flash_8mb.bin")
 if not os.path.isfile(FLASH):
-    raise RuntimeError(f"Flash image not found: {FLASH}")
+    raise RuntimeError(f"Flash image not found: {FLASH}. Run: dd if=/dev/zero bs=1M count=8 of=build/qemu_flash_8mb.bin && dd if=build/qemu_flash.bin of=build/qemu_flash_8mb.bin conv=notrunc")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)